frontend/src/pages/off-nav/ResetPasswordPage.js

```diff
@@ -133,6 +145,7 @@ export default function ResetPasswordPage({ navigation, route }) {
       <TouchableOpacity
         style={styles.button}
         disabled={!fieldsValid()}
+        onPress={() => reset()}
       >
         <Text style={styles.buttonText}>
           RESET
```

Comment:
Apparently the backend doesn't support the Reset Password functionality anymore. Are we still using this form?